frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -57,6 +57,7 @@ const handleError = (
     const errorMessageKey = `api_errors.${errorCode}`;
 
     if (i18n.exists(errorMessageKey)) {
+      // THIS IS CALLED 100 TIMES
```

Comment:
Er dette noe vi burde lage en sak på?

Reply:
Dette er meg som kommenterer koden for å huske igjen hvor feil skjer underveis 😂 Den skal ikke være der, og saken er fikset i denne PRen 😄